webapp/components/FilterPosts/FilterPosts.vue

```diff
@@ -0,0 +1,282 @@
+                  </ds-flex-item>
+                  <ds-space margin-bottom="xx-large" />
+                </ds-flex>
+              </ds-flex>
+            </ds-flex>
+          </ds-flex-item>
+          <ds-flex-item :width="{ base: '50%', sm: '0%', md: '50%', lg: '10%' }">
+            <ds-flex v-for="category in chunk[6]" :key="category.id" class="categories-list">
+              <ds-flex>
+                <ds-flex-item width="100%">
+                  <ds-button
+                    :icon="category.icon"
+                    :primary="isActive(category.id)"
+                    @click.stop.prevent="toggleCategory(category.id)"
+                  />
+                  <ds-space margin-bottom="small" />
+                </ds-flex-item>
+                <ds-flex>
+                  <ds-flex-item>
+                    <label>{{ category.name }}</label>
+                  </ds-flex-item>
+                  <ds-space margin-bottom="xx-large" />
+                </ds-flex>
+              </ds-flex>
+            </ds-flex>
+          </ds-flex-item>
+          <ds-flex-item :width="{ base: '50%', sm: '0%', md: '50%', lg: '10%' }">
+            <ds-flex v-for="category in chunk[7]" :key="category.id" class="categories-list">
+              <ds-flex>
+                <ds-flex-item width="100%">
+                  <ds-button
+                    :icon="category.icon"
+                    :primary="isActive(category.id)"
+                    @click.stop.prevent="toggleCategory(category.id)"
+                  />
+                  <ds-space margin-bottom="small" />
+                </ds-flex-item>
+                <ds-flex>
+                  <ds-flex-item>
+                    <label>{{ category.name }}</label>
+                  </ds-flex-item>
+                  <ds-space margin-bottom="xx-large" />
+                </ds-flex>
+              </ds-flex>
+            </ds-flex>
+          </ds-flex-item>
+        </ds-flex>
+      </ds-container>
+    </template>
+  </dropdown>
+</template>
+<script>
+import Dropdown from '~/components/Dropdown'
+import { filterPosts } from '~/graphql/PostQuery.js'
+import { mapMutations } from 'vuex'
+
+export default {
+  components: {
+    Dropdown,
+  },
+  props: {
+    placement: { type: String, default: 'bottom-start' },
+    offset: { type: [String, Number], default: '16' },
+    categories: { type: Array, default: () => [] },
+  },
+  data() {
+    return {
+      pageSize: 12,
+      selectedCategoryIds: [],
+      allCategories: true,
+    }
+  },
+  computed: {
+    chunk() {
+      const groupedCategories = []
+      for (let i = 0; i < this.categories.length; i++) {
+        const last = groupedCategories[groupedCategories.length - 1]
+        if (!last || last.length === 2) {
+          groupedCategories.push([this.categories[i]])
+        } else {
+          last.push(this.categories[i])
+        }
+      }
+      return groupedCategories
+    },
+  },
```

Comment:
we could use `_.chunk` from lodash for this, if anyone thinks it's better/necessary

Reply:
There is plenty of `lodash` in `yarn.lock`. I wouldn't mind to add it to `package.json`.

Reply:
great